selfdrive/boardd/panda.cc

```diff
@@ -188,6 +188,10 @@ void Panda::set_safety_model(cereal::CarParams::SafetyModel safety_model, int sa
   usb_write(0xdc, (uint16_t)safety_model, safety_param);
 }
 
+void Panda::set_unsafe_mode(int unsafe_mode) {
+  panda->usb_write(0xdf, unsafe_mode, 0);
```

Comment:
Did I get this right? The second argument is the `wValue` which panda/main.cc uses to set unsafe_mode